bears/hypertext/HTMLHintBear.py

```diff
@@ -0,0 +1,179 @@
+            For example: If set to ``underline``, prefer
+            ``<div id="aaa_bbb">``.
+            For example: If set to ``dash``, prefer ``<div id="aaa-bbb">``.
+        :param prohibit_inline_style:
+            Disallow the use of inline ``style`` attribute.
+            For example: If set to ``True``, ``<div style="color:red"></div>``
+            will raise a warning.
+        :param require_relative_links_in_href:
+            If ``True``, enforce relative links in the ``href`` attribute and
+            if ``False``, enforce absolute links.
+        :param prohibit_unsafe_characters:
+            Prohibit the use of unsafe characters in attribute values.
+            For example: If set to ``True``,
+            ``<li><a href="https://vimeo.com//56931059‎\u0009‎">2012</a></li>``
+            will raise a warning.
+        :param prohibit_inline_script:
+            Disallow the use of inline scripts.
+            For example: If set to ``True``, this will raise a warning::
+
+                <img src="test.gif" onclick="alert(1);">
+                <img src="javascript:alert(1)">
+                <a href="javascript:alert(1)">test1</a>
+
+        :param prohibit_style_tag:
+            Prohibit the use of ``style`` tag.
+            For example: If set to ``True``,
+            ``<body><style type="text/css"></style></body>``
+            will raise a warning.
+        """
+        if htmlhint_config:
+            return None
+        else:
+            options = {
+                'tagname-lowercase': enforce_lowercase_tagname,
+                'attr-lowercase': enforce_lowercase_attribute,
+                'attr-value-double-quotes':
+                    require_attribute_value_in_double_quotes,
+                'attr-value-not_empty': prohibit_empty_value_for_attribute,
+                'attr-no-duplication': prohibit_attribute_duplication,
+                'doctype-first': require_doctype_at_beginning,
+                'tag-pair': enforce_tag_pair,
+                'tag-self-close': enforce_self_close_empty_tag,
+                'spec-char-escape': require_escaped_special_characters,
+                'id-unique': require_unique_attribute_id,
+                'title-require': require_title_tag,
+                'head-script-disabled': prohibit_script_in_head,
+                'alt-require': require_alt_attribute,
+                'id-class-value': enforce_id_class_naming_convention,
+                'inline-style-disabled': prohibit_inline_style,
+                'attr-unsafe-chars': prohibit_unsafe_characters,
+                'inline-script-disabled': prohibit_inline_script,
+                'style-disabled': prohibit_style_tag
+            }
+            if require_relative_links_in_href:
+                options['href-abs-or-rel'] = 'rel'
+            elif require_relative_links_in_href is False:
+                options['href-abs-or-rel'] = 'abs'
+            else:
+                options['href-abs-or-rel'] = 'false'
```

Comment:
hm maybe inline it above doing like this:
```python
'href-abs-or-rel': 'rel' if require_relative_links_in_href else
                   'abs' if require_relative_links_in_href is False else
                   'false'
```
As the assignment is repeated here, we should try to avoid that :+1: